_includes/templates/summaries/linkedin.html

```diff
@@ -0,0 +1,16 @@
+<h2 class="h4">
+    <span class="cf-icon cf-icon-linkedin-square"></span>
+    Follow us on LinkedIn
```

Comment:
Yes--should say "Follow us on LinkedIn"

On Tue, Jun 30, 2015 at 3:39 PM, Ans <notifications@github.com> wrote:

> In _includes/templates/summaries/linkedin.html
> <https://github.com/cfpb/cfgov-refresh/pull/686#discussion_r33614544>:
>
> > @@ -0,0 +1,16 @@
> > +<h2 class="h4">
> > +    <span class="cf-icon cf-icon-linkedin-square"></span>
> > +    Follow us on LinkedIn
>
> @sarahsimpson09 <https://github.com/sarahsimpson09> Should it be Follow
> Us On LinkedIn?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cfpb/cfgov-refresh/pull/686/files#r33614544>.
>